.github/workflows/force-docs-build.yml

```diff
@@ -0,0 +1,62 @@
+name: Manual Docs Build
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_token:
+        description: 'Your release token'
+        required: true
+      triggered_by:
+        description: 'CD | TAG | MANUAL'
+        required: false
+        default: MANUAL
+
+jobs:
+  token-check:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "success!"
+        if: "${{ github.event.inputs.release_token }} == ${{ env.release_token }}"
+        env:
+          release_token: ${{ secrets.FINETUNER_RELEASE_TOKEN }}
+
+  release-docs:
+    needs: token-check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Build doc and push to gh-pages
+        run: |
+          git config --local user.email "dev-bot@jina.ai"
+          git config --local user.name "Jina Dev Bot"
+          pip install .
+          mkdir gen-html
+          cd docs
+          pip install -r requirements.txt
+          pip install --pre -U furo
```

Comment:
why do you install furo separately? Shouldn't it be specifified in the requirements.txt you installed just before.

Reply:
not sure, copied from main branch, this is not tested, will clear it up when we merge finetuner client to main branch